x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -82,14 +82,20 @@ export const EnginesList: React.FC = () => {
   const throttledSearchQuery = useThrottle(searchQuery, INPUT_THROTTLE_DELAY_MS);
 
   useEffect(() => {
-    fetchEngines();
+    // Don't fetch engines if we don't have a valid license
+    if (!isGated!) {
```

Comment:
assuming this should be `!isGated` not `!isGated!` ?
```suggestion
    if (!isGated) {
```

Reply:
🤦 Thanks for catching this Rodney!